src/Template/Bake/Template/index.ctp

```diff
@@ -75,9 +75,9 @@ $fields = collection($fields)
             $pk = '$' . $singularVar . '->' . $primaryKey[0];
             %>
             <td class="actions">
-                <?= $this->Html->link(__('View'), ['action' => 'view', <%= $pk %>]) ?>
-                <?= $this->Html->link(__('Edit'), ['action' => 'edit', <%= $pk %>]) ?>
-                <?= $this->Form->postLink(__('Delete'), ['action' => 'delete', <%= $pk %>], ['confirm' => __('Are you sure you want to delete # {0}?', <%= $pk %>)]) ?>
+                <?= $this->Html->link('', ['action' => 'view', <%= $pk %>], ['title' => __('View'), 'alt' => __('View'), 'class' => 'btn btn-default glyphicon glyphicon-eye-open']) ?>
```

Comment:
Right, those are not img, but font icons using a/span/i tags. So alt is not necessary after all. In case the font cannot be loaded, what is the "alt" info displayed then?